subsys/net/l2/wifi/Kconfig

```diff
@@ -93,3 +93,12 @@ module-str = Log level for Wi-Fi Network manager module
 module-help = Enables using the Wi-Fi Network managers to manage the Wi-Fi network interfaces.
 source "subsys/net/Kconfig.template.log_config.net"
 endif # WIFI_NM
+
+config WPA_SUPP_AP_STA_INACTIVITY_TIMEOUT
```

Comment:
Was this added by mistake here? I don't think wpa supplicant-prefixed configs belong here, plus it doesn't seem to be used anywhere.